### PR TITLE
28 installation document

### PR DESCRIPTION
**Pull request by zafertaskent**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:latest
```
Comment (koc-university) [inline Dockerfile]: Remove this file for now

File: config/env/development.js [hunk 2]
```diff
@@ -20,15 +20,5 @@ module.exports = {
   // models: {
   //   connection: 'someMongodbServer'
   // }
-  host: '127.0.0.1',
-  port: 1337,
-  google: {
```
Comment (koc-university) [inline config/env/development.js]: If we do this change development might stop. @ogzhnozlci @berkantesk  change your config/local.js according to this document.

File: config/env/development.js [hunk 3]
```diff
@@ -20,15 +20,5 @@ module.exports = {
...
-    clientID: '412383755015-qs4k8eevr0e0nobo7monck1q5drjeupm.apps.googleusercontent.com',
```
Comment (koc-university) [inline config/env/development.js]: @berkantesk @ogzhnozlci basically you should copy these to config/local.js

**Comment by zafertaskent** [comment]
Dockerfile is removed

**Pull request merged**